### PR TITLE
[ES] Update `BBCode`

### PR DESCRIPTION
**Pull request by UnknownOsu**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/BBCode/es.md [hunk 1]
```diff
@@ -4,411 +4,329 @@ outdated_translation: true
...
+
+```
+[code]
+texto
+[/code]
+```
+
+La etiqueta `[code]` se usa para crear *bloques de código preformateados* (también conocido como *texto preformateado*). En el sitio web de osu!, la etiqueta `[code]` formateará el texto en una fuente monoespaciada dentro de un cuadro gris semitransparente. Dar formato al texto dentro de un bloque de código le indicará al editor que trate el texto entre esas etiquetas literalmente, evitando así la conversión de cualquier etiqueta o código fuente en otra cosa.
+
+Dentro de los foros de osu!, los bloques de código se utilizan con mayor frecuencia para publicar el código fuente de una [storyboard](/wiki/Guión gráfico), o en tutoriales que requieren mostrar la sintaxis de las etiquetas, los comandos o el código fuente.
```
Comment (agatemosu) [inline wiki/BBCode/es.md:174]: google translate

File: wiki/BBCode/es.md [hunk 2]
```diff
@@ -4,411 +4,329 @@ outdated_translation: true
...
+
+### Encabezado (v2)
 
-### Trivia
+```
+[texto]
+```
 
-- Guía original: [HOW TO: Forum BBCodes (inglés)](https://osu.ppy.sh/community/forums/topics/445599) por [Stefan](https://osu.ppy.sh/users/626907)
+La etiqueta *Encabezado (v2)* es una etiqueta desactualizada que alguna vez se usó en los foros de osu! para dar formato al texto en un encabezado púrpura de apariencia más elegante con una línea horizontal. La etiqueta solo funcionó en el foro de Mapas, y solo aparecia después de la publicación (no en la vista previa). No tenía botón cuando estaba en servicio y se indicaba con un corchete abierto y cerrado (sin etiquetas de apertura y cierre).
```
Comment (agatemosu) [inline wiki/BBCode/es.md:326]:
```suggestion
La etiqueta *Encabezado (v2)* es una etiqueta desactualizada que alguna vez se usó en los foros de osu! para dar formato al texto en un encabezado púrpura de apariencia más elegante con una línea horizontal. La etiqueta solo funcionó en el foro Beatmaps, y solo aparecia después de la publicación (no en la vista previa). No tenía botón cuando estaba en servicio y se indicaba con un corchete abierto y cerrado (sin etiquetas de apertura y cierre).
```

File: wiki/BBCode/es.md [hunk 3]
```diff
@@ -4,411 +4,329 @@ outdated_translation: true
...
 
-### Historia
+## Trivialidades
```
Comment (agatemosu) [inline wiki/BBCode/es.md:328]:
```suggestion
## Curiosidades
```

**Review by agatemosu — CHANGES_REQUESTED** [review]

File: wiki/BBCode/es.md [hunk 4]
```diff
@@ -4,411 +4,329 @@ outdated_translation: true
...
-  ```
+<!-- Example online audio file URL for wiki editors: https://actions.google.com/sounds/v1/alarms/digital_watch_alarm_long.ogg -->
+
+### Encabezado (v1)
+
+```
+[heading]texto[/heading]
+```
+
+La etiqueta `[heading]` se utiliza para dar formato al texto en grandes encabezados de color rosa. La etiqueta no admite encabezados de varios niveles y no se puede vincular específicamente.
```
Comment (agatemosu) [inline wiki/BBCode/es.md:284]:
```suggestion
La etiqueta `[heading]` se usa para dar formato al texto en grandes encabezados de color rosa. La etiqueta no admite encabezados de varios niveles y no se puede vincular específicamente.
```

File: wiki/BBCode/es.md [hunk 5]
```diff
@@ -4,411 +4,329 @@ outdated_translation: true
...
 
 ### Perfil
 
-Vincula un perfil de usuario utilizando el nombre del usuario.
+```
+[profile=userid]nombre de usuario[/profile]
+```
 
-¡El uso de este botón **no es recomendable**! El problema con el uso de esta etiqueta es que los nombres de usuario pueden cambiarse una vez obtenida una [tag de osu! Supporter](/wiki/osu!supporter). Y una vez que lo hacen, el enlace fallará.
+La etiqueta `[profile]` se utiliza para enlazar la página de perfil de osu! de un usuario usando su nombre de usuario o ID de usuario. El uso de la etiqueta `[profile]` difiere del uso de la etiqueta `[url]` en que la etiqueta `[profile]` muestra una tarjeta de usuario al pasar el cursor sobre el enlace creado por la etiqueta.
```
Comment (agatemosu) [inline wiki/BBCode/es.md:204]:
```suggestion
La etiqueta `[profile]` se usa para enlazar la página de perfil de osu! de un usuario usando su nombre de usuario o ID de usuario. El uso de la etiqueta `[profile]` difiere del uso de la etiqueta `[url]` en que la etiqueta `[profile]` muestra una tarjeta de usuario al pasar el cursor sobre el enlace creado por la etiqueta.
```

**Review by agatemosu — CHANGES_REQUESTED** [review]

File: wiki/BBCode/es.md [hunk 6]
```diff
@@ -4,411 +4,329 @@ outdated_translation: true
...
+La etiqueta *Encabezado (v2)* es una etiqueta desactualizada que alguna vez se usó en los foros de osu! para dar formato al texto en un encabezado púrpura de apariencia más elegante con una línea horizontal. La etiqueta solo funcionó en el foro de Mapas, y solo aparecia después de la publicación (no en la vista previa). No tenía botón cuando estaba en servicio y se indicaba con un corchete abierto y cerrado (sin etiquetas de apertura y cierre).
 
-### Historia
+## Trivialidades
 
-- Hay un error de color en los foros actuales de la web de osu!, si usas el color `transparent`, hará que el texto sea transparente.
-  - Este error fue sin embargo, resuelto en el nuevo diseño.
+- Este artículo de la wiki fue adaptado del hilo del foro ["HOW TO: Forum BBCodes"](https://osu.ppy.sh/community/forums/topics/445599) por [Stefan](https://osu.ppy.sh/users/626907).
+- Solía haber un error que permitía a los usuarios hacer que el texto fuera transparente usando la [etiqueta de color](#color) y escribiendo "transparente" después del signo igual (`=`).
```
Comment (agatemosu) [inline wiki/BBCode/es.md:331]:
```suggestion
- Solía haber un error que permitía a los usuarios hacer que el texto fuera transparente usando la [etiqueta de color](#color) y escribiendo «transparent» después del signo igual (`=`).
```

File: wiki/BBCode/es.md [hunk 7]
```diff
@@ -4,411 +4,329 @@ outdated_translation: true
...
 
-- Guía original: [HOW TO: Forum BBCodes (inglés)](https://osu.ppy.sh/community/forums/topics/445599) por [Stefan](https://osu.ppy.sh/users/626907)
+La etiqueta *Encabezado (v2)* es una etiqueta desactualizada que alguna vez se usó en los foros de osu! para dar formato al texto en un encabezado púrpura de apariencia más elegante con una línea horizontal. La etiqueta solo funcionó en el foro de Mapas, y solo aparecia después de la publicación (no en la vista previa). No tenía botón cuando estaba en servicio y se indicaba con un corchete abierto y cerrado (sin etiquetas de apertura y cierre).
 
-### Historia
+## Trivialidades
 
-- Hay un error de color en los foros actuales de la web de osu!, si usas el color `transparent`, hará que el texto sea transparente.
-  - Este error fue sin embargo, resuelto en el nuevo diseño.
+- Este artículo de la wiki fue adaptado del hilo del foro ["HOW TO: Forum BBCodes"](https://osu.ppy.sh/community/forums/topics/445599) por [Stefan](https://osu.ppy.sh/users/626907).
```
Comment (agatemosu) [inline wiki/BBCode/es.md:330]:
```suggestion
- Este artículo de la wiki fue adaptado del hilo del foro [«HOW TO: Forum BBCodes»](https://osu.ppy.sh/community/forums/topics/445599) por [Stefan](https://osu.ppy.sh/users/626907).
```

File: wiki/BBCode/es.md (hunk 3 above)
Comment (agatemosu) [inline wiki/BBCode/es.md:328]:
```suggestion
## Curiosidades
```

**Review by agatemosu — CHANGES_REQUESTED** [review]

**Comment by The-Last-Cookie** [comment]
Keep in mind translating the new `Help` string in the toolbar via Crowdin and updating the `editor-ES.jpg` file.

**Comment by UnknownOsu** [comment]
I believe it is not yet available for translation, but I may be wrong.

The image I think it is better to update it when the button is already translated to Spanish, I think it would not do much good to update it when the button is still in English.

For the moment I update the document with the new tag that was added (`imagemap`). 

Let me know if I did something wrong, I will change it as soon as possible.

**Comment by The-Last-Cookie** [comment]
> I believe it is not yet available for translation, but I may be wrong.

Yes, but it should be available anytime soon (probably next i18n build). Sorry that I didn't mention this.

> The image I think it is better to update it when the button is already translated to Spanish, I think it would not do much good to update it when the button is still in English.

That is precisely what I wanted to notify you of with my comment. I mentioned it, so updating the image is not forgotten. When viewing the diff on the osu wiki status page, the image update is not visible as far as I know.

> Let me know if I did something wrong, I will change it as soon as possible.

No worries! Everything's alright.

**Comment by UnknownOsu** [comment]
> Yes, but it should be available anytime soon (probably next i18n build). Sorry that I didn't mention this.

 Don't worry
  
> That is precisely what I wanted to notify you of with my comment. I mentioned it, so updating the image is not forgotten. When viewing the diff on the osu wiki status page, the image update is not visible as far as I know.

Yeah, the old translation did not show the translated image. In this one I did change it, but now I have to wait to update it to the latest one.

**Comment by agatemosu** [comment]
The image should be like this right?
editor-ES

**Comment by UnknownOsu** [comment]
Thanks for the idea, agatem.

**Comment by TicClick** [comment]
@UnknownOsu TODO:
- please resolve the conflicts (choose which fragments from the conflicting ones should be present in your final version of the translation) using the `Resolve conflicts` button
- please address @agatemosu's review

**Comment by UnknownOsu** [comment]
I think that's it, although I don't know what you mean by the last point to do. ;(

**Comment by TicClick** [comment]
> I think that's it, although I don't know what you mean by the last point to do. ;(

https://github.com/ppy/osu-wiki/pull/9141#pullrequestreview-1381424510 onwards -- either edit your changes accordingly, or state why you aren't going to

**Comment by UnknownOsu** [comment]
> [#9141 (review)](https://github.com/ppy/osu-wiki/pull/9141#pullrequestreview-1381424510) onwards -- either edit your changes accordingly, or state why you aren't going to

So, should I accept the suggestions that I think are right with the button that pops up there, and explain the ones that I think are wrong?

**Comment by TicClick** [comment]
yeah

File: wiki/BBCode/es.md [hunk 8]
```diff
@@ -4,411 +4,329 @@ outdated_translation: true
...
+
+```
+[img]ADDRESS[/img]
+```
+
+La etiqueta `[img]` se usa para incluir imágenes en línea en las publicaciones de foro de osu!. Para usar la etiqueta, los usuarios deben pegar la dirección de la imagen directa (representada por el argumento `ADDRESS` anterior) procedente de un sitio web. Las rutas de archivos locales (por ejemplo, `C:\Users\Name\Pictures\image.jpg`) **no funcionarán**.
+
+*Aviso: la URL de un sitio web **no** es lo mismo que la dirección de una imagen.*
+
+Para obtener una dirección de imagen, uno debe navegar al sitio web en el que se encuentra, pasar el mouse sobre la imagen, hacer clic con el botón derecho en la imagen y seleccionar `Copiar dirección de la imagen`. Luego, la dirección debe copiarse y pegarse entre las etiquetas.
```
[the remark this answers is not in the document]
Comment (UnknownOsu) [reply wiki/BBCode/es.md:248]: Creo que en la mayoría de lenguajes en español sale [`Copiar dirección de la imagen`](https://i.imgur.com/eH55DTV.png).

```suggestion
Para obtener una dirección de imagen, uno debe navegar al sitio web en el que se encuentra, pasar el ratón sobre la imagen, hacer clic con el botón derecho en la imagen y seleccionar `Copiar dirección de la imagen`. Luego, la dirección debe copiarse y pegarse entre las etiquetas.
```

File: wiki/BBCode/es.md [hunk 9]
```diff
@@ -4,411 +4,329 @@ outdated_translation: true
...
-- Notas:
-  - ¡Esto sólo funciona en el foro de beatmaps!
-  - Esto sólo aparece después de la publicación, no en la vista previa.
-  - Debes escribir la sintaxis tú mismo.
-- Sintaxis:
-  ```
-  [...]
-  ```
+<!-- Example online audio file URL for wiki editors: https://actions.google.com/sounds/v1/alarms/digital_watch_alarm_long.ogg -->
+
```
[the remark this answers is not in the document]
Comment (UnknownOsu) [reply wiki/BBCode/es.md:292]: El audio funciona y no interfiere en nada con el artículo.

**Comment by UnknownOsu** [comment]
Well, I think this article is ready. I'll check the other one now.

File: wiki/BBCode/es.md [hunk 10]
```diff
@@ -4,411 +4,345 @@ outdated_translation: true
...
+[quote="NOMBRE"]
+texto
+[/quote]
+```
+
+La etiqueta `[quote]` se usa para dar formato estilístico a citas largas (también conocidas como «citas en bloque») mediante el uso de sangría, color, negrita y separación del texto a través de una línea vertical rosa. El contenido real de la cita se coloca entre las etiquetas abiertas y cerradas, mientras que el argumento `NOMBRE` especifica el autor de la cita (aunque esto es opcional). El texto dentro de las comillas generará espacios en blanco y saltos de línea.
+
+*Aviso: el argumento `NOMBRE` debe estar entre comillas (`"`).*
+
+Las citas largas generalmente se usan en escritos más formales en lugar de citas en línea cuando dicha cita tiene tres o más líneas. Sin embargo, dentro de los foros de osu!, se usan más comúnmente para responder al comentario de otro usuario, lo que se puede hacer automáticamente a través del botón `Citar y responder` ubicado en la parte superior derecha del comentario deseado (que se muestra a continuación). Sin embargo, este botón **solo aparecerá si el cursor está cerca**.
```
[the remark this answers is not in the document]
Comment (UnknownOsu) [reply wiki/BBCode/es.md:146]: Puse [`Citar y responder`](https://i.imgur.com/SuI9pWn.png) porque el botón dice eso al momento de querer responder a un comentario. Sale algo más, pero en el original tampoco pusieron todo el texto.

A menos que se refiera a otro botón, creo que está bien.

File: wiki/BBCode/es.md [hunk 11]
```diff
@@ -4,411 +4,344 @@ outdated_translation: true
...
-  - ¡Esto solo funciona en el foro de beatmaps!
-  - Esto solo aparece después de la publicación, no en la vista previa.
-  - Debes escribir la sintaxis tú mismo.
-- Sintaxis:
-  ```
-  [...]
-  ```
+Para incrustar archivos de audio a través de este método, los usuarios deben pegar su URL de origen (por ejemplo, `https://www.example.com/example.mp3`) entre las dos etiquetas `[audio]`.
+
+
```
Comment (agatemosu) [inline wiki/BBCode/es.md:291]:
```suggestion
```

**Review by agatemosu — APPROVED** [review]

File: wiki/BBCode/es.md [hunk 12]
```diff
@@ -4,411 +4,343 @@ outdated_translation: true
...
+URL_DE_LA_IMAGEN
+X Y ANCHO ALTURA REDIRECCIÓN TÍTULO
+[/imagemap]
+```
+
+La etiqueta `[imagemap]` se utiliza para integrar uno o más hipervínculos en una imagen en áreas rectangulares.
+
+La imagen, que está incrustada en el sitio web, está representada por el argumento `URL_DE_LA_IMAGEN`. Debe referirse directamente a una imagen alojada en un sitio web.
+
+Para agregar un área en la que se puede hacer clic, se debe insertar una nueva línea que contenga la posición x e y del área, el ancho y el alto del área, así como un enlace para redirigir después del argumento `URL_DE_LA_IMAGEN`. Además, se mostrará un argumento `TÍTULO` opcional al pasar el cursor por el área si se especifica. Un enlace se puede especificar con el argumento `REDIRECCIÓN` u omitirse con `#`. Cada unidad de tamaño (`X`, `Y`, `ANCHO` y `ALTURA`) es un porcentaje (0–100) sin el signo de porcentaje.
```
Comment (TPGPL) [inline wiki/BBCode/es.md:263]: `Toolbar button: Imagemap button` line is missing

Comment (agatemosu) [reply wiki/BBCode/es.md:263]: Botón de la barra de herramientas: Botón de mapa de imagen

File: wiki/BBCode/es.md [hunk 13]
```diff
@@ -1,414 +1,342 @@
...
```
Comment (agatemosu) [inline wiki/BBCode/es.md:264]:
```suggestion

Botón de la barra de herramientas: ![Botón de mapa de imagen](img/imagemap.png "Mapa de imagen")
```

**Review by agatemosu — CHANGES_REQUESTED** [review]

**Review by TPGPL — APPROVED** [review]

**Pull request merged**